examples/stm32/f0/other/can/can.c

```diff
@@ -0,0 +1,182 @@
+/*
+ * This file is part of the libopencm3 project.
+ *
+ * Copyright (C) 2010 Thomas Otto <tommi@viadmin.org>
+ * Copyright (C) 2010 Piotr Esden-Tempski <piotr@esden.net>
```

Comment:
I don't belive this to be the case.  Either use your own name, omit the claim entirely, or demonstrate what parts they wrote.